spec/Section 5 -- Validation.md

```diff
@@ -780,6 +780,20 @@ fragment missingRequiredArg on Arguments {
 }
 ```
 
+#### Oneof Fields Have Exactly One Argument
+
+* For each Oneof Field in the document:
+  * Let {arguments} be the arguments provided by the Field.
+  * {arguments} must contain exactly one entry.
+  * For the sole {argument} in {arguments}:
+    * Let {value} be the value of {argument}.
+    * {value} must not be the {null} literal.
+
+**Explanatory Text**
+
+Oneof Fields require that exactly one argument must be supplied and that
```

Comment:
Does this mean that the validation rule should check the **actual** values of the request variables? As far as I know now, none of the validation rules need to check actual variables values.

Reply:
Yes, the input object must be validated (via coercion) at runtime, same as regular input objects. We just expand that logic with oneOf-specific details. See: https://spec.graphql.org/draft/#sel-FAHhBVJCAACIBxnC